pandas/core/indexes/multi.py

```diff
@@ -1647,7 +1647,10 @@ def get_level_values(self, level):
         """
         Return vector of label values for requested level.
 
-        Length of returned vector is equal to the length of the index.
+        Length of returned vector is equal to the length of the index. Note
```

Comment:
can you make this a Note itself, e.g. the added text.  (e.g. sub-section).

Reply:
Done! @jreback